h2o-r/h2o-package/R/gam.R

```diff
@@ -117,7 +117,7 @@
 #' prostate <- h2o.uploadFile(path = prostate_path)
 #' prostate$CAPSULE <- as.factor(prostate$CAPSULE)
 #' h2o.gam(y = "CAPSULE", x = c("RACE"), gam_columns = c("PSA"),
-#'      training_frame = prostate,family = "binomial")
+#'      training_frame = prostate, family = "binomial")
```

Comment:
@hannah-tillman this is the wrong file to update. The changes to this and other algos have to be made in:
/h2o-3/h2o-bindings/bin/custom/R

Reply:
Oh man... I have done this so many times... Sorry @hannah-tillman.  Thanks @angela0xdata for pointing this out. :-(